stake-pool/cli/src/main.rs

```diff
@@ -2,9 +2,9 @@
 mod client;
 mod output;
 
-// use instruction::create_associated_token_account once ATA 1.0.5 is released
-#[allow(deprecated)]
-use spl_associated_token_account::create_associated_token_account;
+use spl_associated_token_account::instruction::create_associated_token_account; // once ATA 1.0.5 is released
```

Comment:
This is great, thanks! Normally I'd ask for it to go in a separate PR, but it's a small enough change. You could even remove the comment
```suggestion
use spl_associated_token_account::instruction::create_associated_token_account;
```